apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -191,12 +191,12 @@ public function register_blocks() {
 		register_block_type(
 			'a8c/navigation-menu',
 			array(
-				'attributes'      => [
-					'className' => [
+				'attributes'      => array(
+					'className' => array(
 						'default' => '',
 						'type'    => 'string',
-					],
-				],
+					),
+				),
```

Comment:
Why all the janitorial changes here?

Reply:
There doesn't appear to be any other changes to the Full Site Editing plugin? Do you mind dropping the related commits?

Reply:
+1 For dropping these changes, we've been using array shorthand syntax throughout the FSE plugin on purpose. For context, our minimum plugin requirements are PHP 5.6+.

Reply:
Thanks @gwwar and @vindl, that change should not have been included in this draft – it's now removed.